src/openloco/window.h

```diff
@@ -408,16 +325,22 @@ namespace openloco::ui
         int16_t var_88C;
 
         window(gfx::point_t position, gfx::ui_size_t size);
-        constexpr void set_size(gfx::ui_size_t size)
+
+        constexpr void set_size(gfx::ui_size_t minSize, gfx::ui_size_t maxSize)
```

Comment:
I'd be fine with this method, if it also did the clamping of the current size. If you don't want to do that, the method name needs to be changed.